modules/core/core.mak

```diff
@@ -8,7 +8,7 @@ CPP_LANGUAGE = 14
 
 STRICT = 1
 
-INCLUDEDIRS = $(ROOT_DIR)/modules/core/src/headers $(ICUDIR)/include $(ROOT_DIR)/modules/base/src/headers $(ROOT_DIR)/modules/ali $(ROOT_DIR)/shared/System/unix $(ROOT_DIR)/shared/System $(ROOT_DIR)/shared/Utility $(ROOT_DIR)/kernel/common/h
+INCLUDEDIRS = $(ROOT_DIR)/modules/shell/src $(ROOT_DIR)/modules/core/src/headers $(ICUDIR)/include $(ROOT_DIR)/modules/base/src/headers $(ROOT_DIR)/modules/ali $(ROOT_DIR)/shared/System/unix $(ROOT_DIR)/shared/System $(ROOT_DIR)/shared/Utility $(ROOT_DIR)/kernel/common/h
```

Comment:
core should not have any dependencies on shell, the relationship goes in the other direction. Why was this necessary?

Reply:
I do not know why theoretically necessary, but on Ubuntu running the make without this addition there was a compile error about a missing include.  It could be that there are files in GitHub that have been deleted from Perforce causing this compile error.

Reply:
What's the error?

Reply:
I did a make clean, and then reran the make after restoring core.mak to its original state.  The make succeeded.  Probably Jos corrected the problem.  So the change to core.mak can be backed off.  I will back it off after I understand how in GitHub one backs off a change to a single file.

Reply:
I removed the change locally on jslevinson-cloud/iknow.  I then committed and pushed the change.  The core.mak makefile should be back to its original state post whatever changes were made in intersystems/iknow.

Reply:
I wrote to you earlier that I did a make clean and then make all succeeded.  So changes subsequent to my change to core.mak must have resolved the problem.